docs/swagger.yaml

```diff
@@ -2003,7 +2003,6 @@ components:
       required:
         - projectId
         - userId
-        - status
```

Comment:
I believed that, to be consistent, in the `PUT` operation `status` should have a default value like in the  `POST` operation. So `status` is no longer being a required field.

Reply:
Good catch, agree 👍